app/routes/_dashboard_.courses_.ttc2022.tsx

```diff
@@ -0,0 +1,116 @@
+import type { LoaderArgs, SerializeFrom } from "@remix-run/node";
+import { json } from "@remix-run/node";
+import { NavLink, useLoaderData, useRouteError } from "@remix-run/react";
+import { getContentById } from "~/models/content.server";
+import invariant from "tiny-invariant";
+import Level from "~/components/Level";
+import type { Content, Tag } from "@prisma/client";
+import SecondaryButton from "~/components/buttons/SecondaryButton";
+import CheckTask from "~/components/CheckTask";
+import ErrorComponent from "~/components/Error";
+
+export const loader = async () => {
+  const ttc2022 = await getContentById("36ae94e3-10ba-4e4d-b5fe-5d987c9d19df");
```

Comment:
Regner med at denne endres til dynamisk id senere? :)

Reply:
Eller hele routing forsåvidt :D

Reply:
dette er et kurs som er manuelt lagt til. Men det kommer kanskje ikke så tydelig frem. Burde vi ha en kommentar i toppen som sier noe om det kanskje?

Reply:
Hm. Betyr dette at vi må opprette nye routes for hvert kurs?

Reply:
Ja, man må det 😅 Slik løsningen er nå, må man lage en child route for hvert kurs også lime inn ID-en manuelt. Jeg tror vi bør ta en avgjørelse sammen hvordan vi tenker kurs skal fungere, for å se om vi kan komme opp med en enklere løsning!